lgc/patch/PatchInOutImportExport.h

```diff
@@ -82,6 +81,7 @@ class PatchInOutImportExport : public Patch, public llvm::InstVisitor<PatchInOut
   void processOutputExport(llvm::CallInst &callInst);
   void processInOutMessage(llvm::CallInst &callInst);
 
+  void visitCallInstrs(std::vector<llvm::Function *> calleeFuncs);
```

Comment:
The argument could be an `ArrayRef<llvm::Function *>` to make it a bit more generic?